selfdrive/controls/lib/drive_helpers.py

```diff
@@ -13,7 +13,6 @@ class MPC_COST_LAT:
   PATH = 1.0
   LANE = 3.0
   HEADING = 1.0
-  STEER_RATE = 1.0
```

Comment:
meh, I would still keep it. Ideally mpc should be completely independent from the car and sterrateCost should not be defined in interfaces